python/lsst/meas/algorithms/detection.py

```diff
@@ -158,124 +158,31 @@ def setDefaults(self):
-flag @c -d to import @b debug.py from your @c PYTHONPATH; see @ref baseDebug for more about @b debug.py files.
-
-The available variables in SourceDetectionTask are:
-<DL>
-  <DT> @c display
-  <DD>
-  - If True, display the exposure on afwDisplay.Display's frame 0.
-    +ve detections in blue, -ve detections in cyan
-  - If display > 1, display the convolved exposure on frame 1
-</DL>
-
-@section meas_algorithms_detection_Example      A complete example of using SourceDetectionTask
+    """Create the detection task.  Most arguments are simply passed onto pipe.base.Task.
 
-This code is in @link measAlgTasks.py@endlink in the examples directory, and can be run as @em e.g.
-@code
-examples/measAlgTasks.py --doDisplay
-@endcode
-@dontinclude measAlgTasks.py
-The example also runs the SingleFrameMeasurementTask; see @ref meas_algorithms_measurement_Example for more
-explanation.
-
-Import the task (there are some other standard imports; read the file if you're confused)
-@skipline SourceDetectionTask
-
-We need to create our task before processing any data as the task constructor
-can add an extra column to the schema, but first we need an almost-empty Schema
-@skipline makeMinimalSchema
-after which we can call the constructor:
-@skip SourceDetectionTask.ConfigClass
-@until detectionTask
-
-We're now ready to process the data (we could loop over multiple exposures/catalogues using the same
-task objects).  First create the output table:
-@skipline afwTable
-
-And process the image
-@skipline result
-(You may not be happy that the threshold was set in the config before creating the Task rather than being set
-separately for each exposure.  You @em can reset it just before calling the run method if you must, but we
-should really implement a better solution).
-
-We can then unpack and use the results:
-@skip sources
-@until print
-
-<HR>
-To investigate the @ref meas_algorithms_detection_Debug, put something like
-@code{.py}
-    import lsstDebug
-    def DebugInfo(name):
-        di = lsstDebug.getInfo(name)        # N.b. lsstDebug.Info(name) would call us recursively
-        if name == "lsst.meas.algorithms.detection":
-            di.display = 1
-
-        return di
-
-    lsstDebug.Info = DebugInfo
-@endcode
-into your debug.py file and run measAlgTasks.py with the @c --debug flag.
+    Parameters
+    ----------
+    schema : `lsst.afw.table.Schema`
+        Schema object used to create the output lsst.afw.table.SourceCatalog
+    **kwds
+        Keyword arguments passed to `lsst.pipe.base.task.Task.__init__`
+
+    If schema is not None and configured for 'both' detections,
+    a 'flags.negative' field will be added to label detections made with a
+    negative threshold.
+
+    Notes
+    -----
+    This task can add fields to the schema, so any code calling this task must ensure that
+    these columns are indeed present in the input match list; see Example.
```

Comment:
This example doesn't exist here any more, and may have never existed, based on my reading of the doxygen. I think we can just remove `; see Example`.